index.js

```diff
@@ -104,6 +106,7 @@ class NodeClam {
      * @param {boolean} [options.clamdscan.reloadDb=false] - If true, will re-load the DB on ever call (slow)
      * @param {boolean} [options.clamdscan.active=true] - If true, this module will consider using the `clamdscan` binary
      * @param {boolean} [options.clamdscan.bypassTest=false] - If true, check to see if socket is avaliable
+     * @param {boolean} [options.clamdscan.tls=false] - If true, connect to a TLS-Termination proxy in from of ClamAV
```

Comment:
You have a typo in this JSDoc comment. Did you mean "...in _form_ of ClamAV"?

Reply:
Yikes!
I was going for `in front of`

Reply:
That makes a lot more sense, haha.